Src/BootCamp.Chapter/Checks.cs

```diff
@@ -19,29 +18,23 @@ public static class Checks
         
         public static int PromptInt(string message)
         {
-            
-            // To do: call your implementation. 
-            return Convert.ToInt32(PromptString(message)); ;
+            return Convert.ToInt32(PromptString(message));
```

Comment:
Those 4 functions should return something that looks like `Lesson3.Function(message)` :)